README.md

```diff
@@ -93,7 +93,7 @@ To make it discoverable by your Vulkan loader:
      - It's also possible to provide full path to the layer library (e.g. `/home/user/dxvk-nvapi/layer/libdxvk_nvapi_vkreflex_layer.so`)
      - Finally, if the layer library is in one of the paths searched by dynamic linker (like `/usr/lib`, this depends on the distro), it's possible to set the library path to just the name of the layer library: `"library_path": "libdxvk_nvapi_vkreflex_layer.so"`
 
-Because the layer could interfere with other Vulkan applications that don't use Reflex, it is currently _disabled by default_ even when installed. To enable it, export `DXVK_NVAPI_VKREFLEX=1` as environment variable. When an application attempts to use Vulkan Reflex and the layer isn't installed and enabled, the following message should be logged at `info` log level: `info:nvapi64:<-NvAPI_Vulkan_InitLowLatencyDevice: Not supported`, with an explanation directing the user to verify the layer's setup.
+Because the layer could interfere with other Vulkan applications that don't use Reflex, it is currently _disabled by default_ even when installed. To enable it, export `DXVK_NVAPI_VKREFLEX=1` as environment variable. When an application attempts to use Vulkan Reflex and the layer isn't installed and enabled, Reflex API calls will return fake successes (falling back to immediate semaphore signalling via `vkSignalSemaphore`) to workaround issues like pink tint in DOOM: The Dark Ages. In this case, a message will be logged (on `info` log level) explaining that Reflex is not actually functional and suggesting to install the layer.
```

Comment:
An alternative to the last sentence would be:
In this case, Reflex appears to be working for the application, but it is not actually functional and latency will not be reduced. Also a message will be logged, on `info` log level, explaining this and suggesting to install the layer.

Reply:
Well, it's pretty much the same thing, just lighter on technical details :sweat_smile: